benchmarks/serialize.py

```diff
@@ -146,29 +147,26 @@ def run(args: argparse.Namespace) -> None:
         serializer = serializer_factory(hasher, max_workers=args.max_workers)
 
     # 3. Signing layer
-    if args.skip_manifest:
-        in_toto_builder = id  # Do nothing, just evaluate the argument
+    if args.single_digest:
+        in_toto_builder = in_toto.SingleDigestIntotoPayload
     else:
```

Comment:
the diff looks a little weird, but this is just removing this block and moving the indents over:
```diff
-if args.skip_manifest:
-        in_toto_builder = id  # Do nothing, just evaluate the argument
-else:
```

I'm not sure the call to `id` was needed and it messed with type checking

Reply:
That was to not transform the manifest into an in-toto statement. I wanted a function like `lambda x: x` and accidentally used `id` (which is that in Haskell but not in Python). In Python it just returns a number for every objects, related to its memory location, etc., a single identifier.

But it looks like `skip_manifest` is no longer used, so this is good.